app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+import datetime
+import time
+
+while True:
+    timestamp = datetime.datetime.now().strftime("%H_%M_%S")
+
+    with open("app-" + timestamp + ".log", "w") as f:
+        f.write(str(datetime.datetime.now()))
```

Comment:
It's better to store current time to some variable and use it everywhere.
And not to call datetime.datetime.now() more than once.

Reply:
Fixed